tests.py

```diff
@@ -94,6 +96,35 @@ def test_series_generator(self):
             actual = actual.iloc[:nos]
             self.assertTrue(actual.equals(expected))
 
+    def test_channel_type(self):
+        """Channel type"""
+        with dw.open(self.d7dname) as d7d:
+            self.assertFalse(d7d.closed, 'd7d did not open')
+            channel = d7d['ENG_RPM']
+            actual = channel.channel_type
+            expected = 1
+            self.assertEqual(actual, expected)
+
+    def test_channel_index(self):
+        """Channel type"""
+        with dw.open(self.d7dname) as d7d:
+            self.assertFalse(d7d.closed, 'd7d did not open')
+            channel = d7d['ENG_RPM']
+            actual = channel.channel_index
+            expected = 'CAN;0;640'
+            self.assertEqual(actual, expected)
```

Comment:
test_channel_index() fails on my 64 bit linux system in Python 3.5.2. The actual is an empty string ('').
What is your development system?

Reply:
I am developing on Windows 7 64 bit and Python 3.6.2.  If you would consider accepting #11 prior to this PR, then I could rebase this branch onto the `ci` branch and we could see what Travis and AppVeyor say to this.